CONTRIBUTING.md

```diff
@@ -2,29 +2,101 @@
 
 ## Dependencies
 
-We use `poetry` to manage the [dependencies](https://github.com/python-poetry/poetry).
-If you dont have `poetry`, you should install with `make poetry-download`.
+The easiest way to have the environment setup is using conda.
+Conda is the most recommended package manager for scientific
+and data python stack.
 
-To install dependencies and prepare [`pre-commit`](https://pre-commit.com/) hooks you would need to run `install` command:
+If you don't have it installed, you can install it via mambaforge:
+https://github.com/conda-forge/miniforge#mambaforge
+
+Mamba is a fastest layer on top of conda that will fast the way
+conda works.
+
+When you have the conda installed, you can proceed and create
+a new conda environment for epigraphhub.
+
+```bash
+mamba env create --file conda/dev.yaml
+```
+
+This command will create a new conda environment with some development
+dependencies for epigraphhub. In order to proceed, you will need first
+activate your environment:
+
+```bash
+conda activate epigraphhubpy
+```
+
+Now, you can install the epigraphhub dependencies with:
+
+```bash
+poetry install
+```
+
+## Configuration file
+
+`epigraphhub_py` needs a configuration file in order to access the database.
+
+After you have the library installed, you can run the following command
+(just an example):
+
+```bash
+epigraphhub-config \
+  --db-host localhost \
+  --db-port 25432 \
+  --db-credential "public:dev_epigraphhub/dev_epigraph/dev_epigraph" \
+  --db-credential "private:dev_privatehub/dev_epigraph/dev_epigraph" \
+  --db-credential "sandbox:dev_sandbox/dev_epigraph/dev_epigraph"
+```
+We need to have connection for all these 3 databases:
+  - public,
+  - private
+  - sandbox
```

Comment:
I think that it is useful to have things "isolated". In this case, it is possible to have a different credential for each "kind" of database. but also we can have the same for all of them. So the database admin can decide about that.

Reply:
Agree. We should have credentials by type of access. 'private', 'public read-only', and 'public read-write'.

in the future, it is quite possible that we will have more databases, but I don't think we will need different types of access.

Reply:
I changed it a bit, so now the user can create any name for the credential, so it will not limit us to have different approaches in the future. for tests (CI) I am removing private and sandbox as we don't have any specific code for that in this repo.

the test now will use the credential name called dev-epigraphhub, if the user is using the production database locally, it can point to the sandbox